nozzle.py

```diff
@@ -436,23 +441,44 @@ def __call__(self, x_vec, *, time=0, eos, **kwargs):
 
             gamma = eos.gamma()
             gas_const = eos.gas_const()
-            pressure = getIsentropicPressure(mach=self._mach, P0=P0, gamma=gamma)
-            temperature = getIsentropicTemperature(mach=self._mach, T0=T0, gamma=gamma)
+            pressure = getIsentropicPressure(
+                mach=self._mach,
+                P0=P0,
+                gamma=gamma
+            )
+            temperature = getIsentropicTemperature(
+                mach=self._mach,
+                T0=T0,
+                gamma=gamma
+            )
             rho = pressure/temperature/gas_const
 
-            velocity = np.zeros(shape=(self._dim,)) 
+            velocity = np.zeros(shape=(self._dim, ))
```

Comment:
?

Reply:
Here's the parse error causing the linting error you were getting for line 1 of the driver, @anderson2981 .